pkg/operator/k8sutil/configmap.go

```diff
@@ -20,13 +20,16 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"sync"
 	"time"
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
+var loggedOperatorSettings = sync.Map{}
```

Comment:
Is sync map provide some other benefit 

Otherwise we can use
`map[string]string{}`

Reply:
yes, it helps when we try to access data concurrently and avoid any mismatch

Reply:
okay